### PR TITLE
fix: changed to use new catalog endpoint

### PR DESCRIPTION
**Pull request by edewit**
Once https://github.com/fabric8-launcher/ngx-launcher/pull/256 (ngx-forge@0.2.25) is released this change is needed to integrate with the new Catalog endpoint

### CODE REVIEW

**Comment by invincibleJai** [comment]
@edewit is it WIP or dependent on some other PRs I tried linking ngx-forge master with this PR got below errors

screen shot 2018-06-07 at 2 43 11 pm


@arunkumars08 @jyasveer

**Comment by edewit** [comment]
@invincibleJai yes it's dependent on https://github.com/fabric8-ui/fabric8-ui/pull/2980

**Comment by jyasveer** [comment]
@edewit #2980 is merged now. Can you please verify this PR now. Thanks

**Comment by edewit** [comment]
rebased ready to merge

File: package.json [hunk 1]
```diff
@@ -142,7 +142,7 @@
     "ngx-base": "2.3.1",
     "ngx-bootstrap": "1.9.3",
     "ngx-fabric8-wit": "6.20.3",
-    "ngx-forge": "0.2.24",
+    "ngx-forge": "^0.2.28",
```
Comment (joshuawilson) [inline package.json]: You should not have added the carat auto-range. This is a the production application and ranges are locked.

Comment (invincibleJai) [reply package.json]: @joshuawilson have raised a PR with change https://github.com/fabric8-ui/fabric8-ui/pull/3006

**Pull request merged**